administrator/language/de-DE/com_scheduler.ini

```
@@ -84,7 +84,9 @@ COM_SCHEDULER_LAST_RUN_DESC="Letzte Ausführung absteigend"
 COM_SCHEDULER_MANAGER_TASKS="Geplante Aufgaben"
 COM_SCHEDULER_MANAGER_TASK_EDIT="Aufgabe bearbeiten"
 COM_SCHEDULER_MANAGER_TASK_NEW="Neue Aufgabe"
+COM_SCHEDULER_MANAGER_TOOLTIP_NOT_AUTHORIZED="Nicht autorisiert"
```

```suggestion
COM_SCHEDULER_MANAGER_TOOLTIP_NOT_AUTHORIZED="Nicht berechtigt"
```

Da wir an den anderen Stellen ja auch "Berechtigungen" nutzen passt berechtigt besser oder was denkst du?